<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="SourabhKankekar" failures="1" tests="5" name="Test_Edge" time="17.214" errors="0" timestamp="2022-06-21T23:03:27 IST">
  <testcase classname="TestNG.Test_3" name="test_2" time="0.001"/>
  <testcase classname="TestNG.Test_3" name="test_2" time="0.002"/>
  <testcase classname="TestNG.Test_3" name="test_2" time="0.001"/>
  <testcase classname="TestNG.Test_3" name="test_3" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNG.Test_3.test_3(Test_3.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_3 -->
  <testcase classname="TestNG.Test_3" name="test_4" time="0.0">
    <skipped/>
  </testcase> <!-- test_4 -->
  <testcase classname="TestNG.Test_3" name="test_1" time="0.0">
    <ignored/>
  </testcase> <!-- test_1 -->
</testsuite> <!-- Test_Edge -->
